css/style.css

```diff
@@ -0,0 +1,3154 @@
+    left: 250px;
+  }
+
+  .information-inside__iconEllipse {
+    width: 31.18px;
+    height: 31.18px;
+  }
+
+  .information-inside__iconI {
+    width: 5.67px;
+    height: 22.68px;
+    margin-left: -18px;
+    padding-top: 3px;
+  }
+
+  .information-inside__title {
+    font-size: 16px;
+    padding-right: 40px;
+  }
+
+  .description-inside__button {
+    font-size: 16px;
+  }
+
+  .security {
+    display: flex;
+    flex-direction: column-reverse;
+    justify-content: center;
+    position: relative;
+    min-width: 992px;
+    max-width: 1199px;
+  }
+
+  .security__wrap-mobile {
+    min-width: 992px;
+    max-width: 1199px;
+    display: flex;
+    justify-content: center;
+    position: relative;
+  }
+
+  .security__logo {
+    position: relative;
+    padding-top: 10%;
+    width: 70%;
+    z-index: 3;
+    height: 50%;
+  }
+
+  .security__image {
+    position: absolute;
+    padding-top: 0%;
+    right: 10%;
+    width: 90%;
+    z-index: 5;
+    top: 70%;
+  }
+
+  .icon-security__front {
+    font-size: 15px;
+    position: absolute;
+    z-index: 20;
+    left: 24%;
+    top: 53%;
+  }
+
+  .icon-security__rectangleFirst {
+    position: absolute;
+    width: 2.86px;
+    height: 193.79px;
+    left: 23.1%;
+    z-index: 20;
+    top: 62%;
+  }
+
+  .icon-security__ellipseFirst {
+    width: 10.01px;
+    height: 10.01px;
+    position: absolute;
+    left: 22.5%;
+    top: 170%;
+    z-index: 20;
+  }
+
+  .icon-security__side {
+    font-size: 15px;
+    position: absolute;
+    left: 48%;
+    z-index: 20;
+    top: 42%;
+    width: 100px;
+  }
+
+  .icon-security__rectangleSecond {
+    position: absolute;
+    width: 2.86px;
+    height: 87.24px;
+    left: 47%;
+    z-index: 20;
+    top: 50%;
+  }
+
+  .icon-security__ellipseSecond {
+    width: 10.01px;
+    height: 10.01px;
+    position: absolute;
+    left: 46.6%;
+    z-index: 20;
+    top: 98%;
+  }
+
+  .icon-security__rollOver {
+    font-size: 15px;
+    position: absolute;
+    left: 53%;
+    z-index: 20;
+    top: 211%;
+    width: 170px;
+  }
+
+  .icon-security__rectangleThird {
+    width: 2.86px;
+    height: 77.23px;
+    position: absolute;
+    left: 51.7%;
+    z-index: 20;
+    top: 207%;
+  }
+
+  .icon-security__ellipseThird {
+    width: 10.01px;
+    height: 10.01px;
+    position: absolute;
+    left: 51.2%;
+    z-index: 20;
+    top: 205%;
+  }
+
+  .security__description {
+    position: relative;
+    min-width: 992px;
+    max-width: 1199px;
+    width: 100%;
+    padding-bottom: 23%;
+    box-sizing: border-box;
+    margin-top: 40%;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .description-security__title-wrap {
+    width: 50%;
+  }
+
+  .description-security__text {
+    width: 80%;
+  }
+
+  .description-security__wrap-mobile {
+    position: absolute;
+    left: 33%;
+    bottom: 0px;
+    top: 50%;
+    padding-bottom: 50px;
+  }
+
+  .description-security__wrap-ipad {
+    display: none;
+  }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/tesla_project/pullRequest?prid=5520131)